sgl-kernel/csrc/cpu/torch_extension_cpu.cpp

```diff
@@ -20,22 +20,31 @@ limitations under the License.
 #include "shm.h"
 
 // silu_and_mul
-void silu_and_mul_cpu(at::Tensor& out, at::Tensor& input);
+at::Tensor silu_and_mul_cpu(at::Tensor& input);
 
 // rmsnorm
-void rmsnorm_cpu(at::Tensor& output, at::Tensor& input, at::Tensor& weight, double eps);
+at::Tensor rmsnorm_cpu(at::Tensor& input, at::Tensor& weight, double eps);
 
 // fused_add_rmsnorm
 void fused_add_rmsnorm_cpu(at::Tensor& input, at::Tensor& residual, at::Tensor& weight, double eps);
 
 // topk
-void grouped_topk_cpu(at::Tensor& topk_weights, at::Tensor& topk_ids,
-    at::Tensor& hidden_states, at::Tensor& gating_output, int64_t topk,
-    bool renormalize, int64_t num_expert_group, int64_t topk_group);
-
-void biased_grouped_topk_cpu(at::Tensor& topk_weights, at::Tensor& topk_ids,
-    at::Tensor& hidden_states, at::Tensor& gating_output, at::Tensor& correction_bias,
-    int64_t topk, bool renormalize, int64_t num_expert_group, int64_t topk_group);
+std::tuple<at::Tensor, at::Tensor> grouped_topk_cpu(
```

Comment:
follow the same code style in this file:
```C++
std::tuple<at::Tensor, at::Tensor> grouped_topk_cpu(at::Tensor& hidden_states, at::Tensor& gating_output,
    int64_t topk, bool renormalize, int64_t num_expert_group, int64_t topk_group);
```

Reply:
The current style has passed clang-format check. Do you still want to change?

Reply:
Actually, the format check of this repo only check python format. clang-format can check c++ format. Anyway, I have changed to the old form.